index.js

```diff
@@ -97,8 +94,14 @@ const allRules = {
 };
 /* eslint-enable */
 
+const fromEntries = Object.fromEntries || function fromEntries(entries) {
+  const obj = {};
+  for (const [key, value] of entries) obj[key] = value;
+  return obj;
+}
```

Comment:
inlining a polyfill (one that's not matching the spec) is a nonstarter.

Reply:
Why not rewrite it to avoid the expensive polyfill then?

Reply:
The polyfill isn't expensive, it's the cheapest possible way to correctly implement the method.